planutils/package_installation.py

```diff
@@ -193,3 +192,86 @@ def run(target, options):
     if not PACKAGES[target]["runnable"]:
         sys.exit(f"Package {target} is not executable")
     subprocess.run([Path(settings.PLANUTILS_PREFIX) / "packages" / target / "run"] + options)
+
+
+def remote(target, options):
+
+    # 1. check if the target is deployed
+
+    package_url = settings.PAAS_SERVER + "/package"
+    r = requests.get(package_url)
+    remote_packages = r.json()
+
+    remote_package = None
+    for p in remote_packages:
+        if p['package_name'] == target:
+            remote_package = p
+            break
+
+    if (remote_package is None) or ('solve' not in remote_package['endpoint']['services']):
+        sys.exit(f"Package {target} is not remotely deployed")
+
+
+    # 2. unpack the options and target to the right json
+
+    json_options = {}
+    remote_call = remote_package['endpoint']['services']['solve']['call']
+    call_parts = remote_call.split(' ')
+
+    args = {arg['name']: arg for arg in remote_package['endpoint']['services']['solve']['args']}
+
+    if len(options) != len(call_parts)-1:
```

Comment:
need to compare with the number of arguments, not the number of call parts, as some manifests include predefined call arguments.

Reply:
I tried to push a commit fixing this line but I get the following error:

```
$ git push
remote: Permission to AI-Planning/planutils.git denied to nirlipo.
fatal: unable to access 'https://github.com/AI-Planning/planutils.git/': The requested URL returned error: 403
```

Reply:
Permissions fixed.

But I think the comparison is correct. `options` is the command-line options, and `call_parts` are what will be called on the remote. They _should_ match the # of arguments, but there's no guarantee (it's up to the manifest maintainer). Here, we assume that the local call (filling out `options`) will match precisely the remote call.

Reply:
Thanks! Yes, but it should check against number of `args` instead. See line 228, we iterate only over the `arguments` of the `call` defined in the manifest by looking for the `{` and `}` symbols. I'll make the commit so you can test it.

Reply:
Aye, I see the change, but I was just trying to align the command-line call with the remote call string -- should mirror closely, as otherwise there is no way to align. I get that this fixes the `... plan` issue, but I think that may be more of an issue with the package -- should be able to specify the plan name. A more direct regex would be better, but meh...I think this is good enough for now.

#75